addons/hr_holidays/migrations/12.0.1.5/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,242 @@
+---Fields in module 'hr_holidays'---
+hr_holidays  / hr.employee              / current_leave_id (many2one)   : relation is now 'hr.leave.type' ('hr.holidays.status')
+# NOTHING TO DO: handled in the model rename
+
+*** new hr.leave model ***
+# DONE: post-migration: filled from hr.holidays  (when old type is 'remove')
+
+hr_holidays  / hr.holidays              / date_from (datetime)          : now required, default = function
+hr_holidays  / hr.holidays              / date_to (datetime)            : now required, default = function
+DONE: post-migration: filled if empty
+
+hr_holidays  / hr.holidays              / holiday_status_id (many2one)  : relation is now 'hr.leave.type' ('hr.holidays.status')
+hr_holidays  / hr.holidays              / holiday_type (selection)      : selection_keys is now '['category', 'company', 'department', 'employee']' ('['category', 'employee']')
+hr_holidays  / hr.holidays              / linked_request_ids (one2many) : relation is now 'hr.leave' ('hr.holidays')
+hr_holidays  / hr.holidays              / parent_id (many2one)          : relation is now 'hr.leave' ('hr.holidays')
+# NOTHING TO DO
+
+hr_holidays  / hr.leave                 / activity_ids (one2many)       : NEW relation: mail.activity
+hr_holidays  / hr.holidays              / message_last_post (datetime)  : DEL
+hr_holidays  / hr.leave                 / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO: fields inherited from mail mixins
+
+hr_holidays  / hr.leave                 / mode_company_id (many2one)    : NEW relation: res.company
+# NOTHING TO DO: used for new 'company' mode in holiday_type
+
+hr_holidays  / hr.leave                 / request_date_from (date)      : NEW
```

Comment:
You should fill this one and `request_date_to` at least with the date from `date_from` and `date_to`

Reply:
but they are not required :S

Reply:
Of course not at DB level and you won't find any error on migration, but it makes an UI inconsistency when cancelling a leave request and not being able due to required by view fields are not present.